content/70.coi-contribs.md

```diff
@@ -11,3 +11,9 @@
 |Author|Contributions|
 |---|---|{% for author in manubot.authors %}
 |{{author.name}}|{% for contribution in author.contributions %}{{ contribution }}{% if not loop.last %}, {% endif %}{% endfor %}|{% endfor %}
+
+### Acknowledgements
+
+We thank Nick DeVito for assistance with the Evidence-Based Medicine Data Lab COVID-19 TrialsTracker data.
+We are grateful to the following contributors for reviewing pieces of the text: Ipsita Krishnan.
```

Comment:
After Ipsita Krishnan, you can add Dr. Nadia Danilova since she also reviewed/gave small edits for (2) pieces of the text (within adjuvants & within underexplored)